app/controllers/coordinators_controller.rb

```diff
@@ -2,20 +2,34 @@ class CoordinatorsController < ApplicationController
   before_action :ensure_admin!
   before_action :set_coordinator, only: [:show, :edit, :update, :destroy]
 
+  add_breadcrumb 'Home', :root_path
+
   def index
+    add_breadcrumb 'Coordinators'
```

Comment:
The second arg is a link, but the rightmost breadcrumb doesn't need a link since it's the page you're currently on. In fact, even if you pass it a link, it won't render it. So I didn't add links to any of the "current page" crumbs.